core/crypto/signed_hdr.c

```diff
@@ -75,6 +87,9 @@ TEE_Result shdr_verify_signature(const struct shdr *shdr)
 	if (is_weak_hash_algo(hash_algo))
 		goto err;
 
+	if (is_weak_key_size(shdr->algo, ta_pub_key_modulus_size * 8))
```

Comment:
Shouldn't we also have this check in `shdr_verify_signature2()`?

Reply:
> Shouldn't we also have this check in `shdr_verify_signature2()`?

Yes, I'll add the same check there as well.

Reply:
Done!